examples/6.educational_pluto_notebook/simulation-example-solution.jl

```diff
@@ -25,22 +25,22 @@ using KomaMRI, PlutoPlotly, PlutoUI
 
 # ╔═╡ cc66bfed-b61b-4067-8c94-4c54b82a3b42
 md"""
-These are the versions of KomaMRI and their subpackages.
+These are the versions of KomaMRI and its subpackages.
```

Comment:
Revert.

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/253/commits/3fde402ca7a7613045f15f2c77564a4f5491fcfb

Reply:
I want to to be "its"